File: MiniDOMAsteriskLib/CallManagers/Events/DNDState.vb
Imports System.Net.Sockets
Imports System.Threading
Imports System.Net

Namespace CallManagers.Events

    
    Public Class DNDState
        Inherits AsteriskEvent

        Public Sub New()
            MyBase.New("DNDState")
        End Sub

        Public Sub New(ByVal rows() As String)
            MyBase.New(rows)
        End Sub


        Public ReadOnly Property Channel As String
            Get
                Return Me.GetAttribute("Channel")
            End Get
        End Property

        Public ReadOnly Property Status As String
            Get
                Return Me.GetAttribute("Status")
            End Get
        End Property
         
    End Class

End Namespace

File: MiniDOMAsteriskLib/CallManagers/Events/DNDState.cs

namespace minidom.CallManagers.Events
{
    public class DNDState : AsteriskEvent
    {
        public DNDState() : base("DNDState")
        {
        }

        public DNDState(string[] rows) : base(rows)
        {
        }

        public string Channel
        {
            get
            {
                return GetAttribute("Channel");
            }
        }

        public string Status
        {
            get
            {
                return GetAttribute("Status");
            }
        }
    }
}